src/main/java/org/jabref/gui/cleanup/CleanupPresetPanel.java

```diff
@@ -111,6 +116,10 @@ public JPanel getPanel() {
         return panel;
     }
 
+    public JScrollPane getScrollPane() {
+        return scrollPane;
```

Comment:
It is probably better to return `scrollPane` in `getPanel` instead of introducing this new getter.